tpch-scripts/TpchQueryExecute.sh

```diff
@@ -22,18 +21,25 @@ QUERY_DIR=$BENCH_HOME/$BENCHMARK/sample-queries-tpch
 
 RESULT_DIR=$BENCH_HOME/$BENCHMARK/results/
 
+PLAN_DIR=$BENCH_HOME/$BENCHMARK/plans/
+
 if [ ! -d "$RESULT_DIR" ]; then
 mkdir $RESULT_DIR
 chmod -R 777 $RESULT_DIR
 fi
 
+if [ ! -d "$PLAN_DIR" ]; then
+mkdir $PLAN_DIR
+chmod -R 777 $PLAN_DIR
```

Comment:
why chmod? This should not be needed. We should remove the chmod on results dir as well.